services/brig/src/Brig/API/Handler.hs

```diff
@@ -117,6 +121,13 @@ brigErrorHandlers logger reqId =
 -------------------------------------------------------------------------------
 -- Utilities
 
+-- | Run a 'UserSubsystemError' error and lift to the handler type.
+liftUserSubsystemError ::
+  Sem ((P.Error UserSubsystemError) : r) a ->
```

Comment:
why do you want to eliminate this effect here?  shouldn't it be `(Member (P.Error userSubsystemError) r) => Sem r a -> Handler r a`?

Reply:
We improved this by dropping the helper.